mixnet/src/utils.rs

```diff
@@ -0,0 +1,20 @@
+use std::time::Duration;
+
+use rand::thread_rng;
+use rand_distr::{Distribution, Exp};
+
+/// Get a random interval from an exponential distribution.
+pub fn poisson_interval_sec(rate_per_min: f64) -> Duration {
+    // create an exponential distribution
+    let exp = Exp::new(rate_per_min).unwrap();
+    // generate a random value from the distribution
+    let interval_min = exp.sample(&mut thread_rng());
+    // convert minutes to seconds
+    Duration::from_secs_f64(interval_min * 60.0)
+}
+
+/// Get the mean interval in seconds for a Poisson.
+pub fn poisson_mean_interval_sec(rate_per_min: f64) -> Duration {
+    // the mean interval in seconds for a Poisson process
+    Duration::from_secs_f64(1.0 / rate_per_min * 60.0)
+}
```

Comment:
Looks good in general. Approving this with some suggestions.

Please keep in mind that this function is not really necessary for logic actually for now. In the executable spec, this function is used only for testing (we may be able to refactor the executable spec, but anyway). In the future, we may need to use this function to dynamically update several configurations such as lambda or mu. So, I'm okay to keep this function here.

Also, it would be helpful to add a comment in the first method, that why we use exponential distribution for intervals, even though the function name says 'poisson'. It's explained in the spec as a comment. 

Lastly, it would be great to have a testcase to guarantee that the method you implemented work as intended, even though this function is very simple, because we may have regressions in the future.

Reply:
Let's not merge this now until #570 is merged.